### DISCUSSION TITLE
Zabbix Monitoring of Windows Event logs

### DISCUSSION THREAD

**Discussion by vanesen** [zabbix/community-templates#53] [Q&A]
Hi All,

I would like to know whether there is a specific template for Windows Event Log monitoring as I don't get many details with the logon failure item i have created. 

Thanks and Regards,

Vanesen.

**Comment by mtxadmin** [comment]
You can make template for interested EventIDs by yourself.

For instance, see my template for Exchange:
https://github.com/mtxadmin/zabbix/tree/main/Exchange
